docs/source/pre_application/digital_ocean_tutorial.rst

```diff
@@ -0,0 +1,134 @@
+.. _digital-ocean-tutorial:
+
+======================================================================
+Tutorial for Creating a Threshold PRE Testnet Node using Digital Ocean
+======================================================================
+
+In this tutorial we're going to setup a Threshold PRE Node on the rinkeby testnet.
+There are a few pre-requisites before we can get started.
+First, we need to create accounts at `Digital Ocean <https://cloud.digitalocean.com/>`_ and `Infura <https://infura.io>`_.
+We also need at least 40,000T in our wallet.
+There are currently no Threshold faucets but if you ask in the `Discord <https://discord.gg/Threshold>`_ channel, someone will be happy to help you out.
+
+   .. note::
+
+      Ensure that you are using a wallet on the rinkeby testnet, don't use a mainnet address.
+
+
+
+Digital Ocean
+-------------
+All of our Digital Ocean configuration will be done automatically, but there are two local environment variables we need to set in order to make this work:
+- `DIGITALOCEAN_ACCESS_TOKEN` - Your Digital Ocean `access token <https://docs.digitalocean.com/reference/api/create-personal-access-token/>`_`
+- `DIGITAL_OCEAN_KEY_FINGERPRINT` - Your Digital Ocean `key fingerprint <https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-account/>`_.
+
+Follow those two blog posts and either `export` the variables or add them to your `~/.bashrc` file.
+
+
+Infura
+------
+We need a way to interact with both the Ethereum and Polygon testnet networks; Infura makes this easy for us.
+Create a new project at Infura with product type `ETHEREUM`.
+Also,add the Polygon addon to this project.
+We're going to create two more environment variables:
+
+- `INFURA_RINKEBY_URL`
+- `INFURA_MUMBAI_URL`
+  
+In the **Project Settings**, change the `ENDPOINTS` to `RINKEBY` / `POLYGON_MUMBAI`
+Set the above environment variables to the corresponding `https` endpoint.
+
+
+Overall the environment variable process should look something like:
+
+.. code-block:: bash
+    $ export INFURA_RINKEBY_URL=https://rinkeby.infura.io/v3/bd76baxxxxxxxxxxxxxxxxxxxxxf0ff0
+    $ export INFURA_MUMBAI_URL=https://polygon-mumbai.infura.io/v3/bd76baxxxxxxxxxxxxxxxxxxxxxf0ff0
+    $ export DIGITALOCEAN_ACCESS_TOKEN=4ade7a8701xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxbafd23
+    $ export DIGITAL_OCEAN_KEY_FINGERPRINT=28:38:e7xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx:ca:5c
+
+
+Setup Remote Node
+-----------------
+Locally, we will install `NuCypher Ops <https://github.com/nucypher/nucypher-ops>`_ to handle the heavy lifting of setting up a node.
+
+.. code-block:: bash
+    $ git clone git@github.com:nucypher/nucypher-ops.git
+    $ cd nucypher-ops
+    $ pip3 install -e .
```

Comment:
@vepkenez also published to pypi, so these steps can potentially be simplified to `pip install nucypher-ops`